### PR TITLE
feat: Generate streamers list from JSON

### PR DESCRIPTION
**Pull request by gillchristian**
Moved the streamers list to `streamers.json` and added a Stack script to generate the list in the README from the JSON

### CODE REVIEW

**Comment by gillchristian** [comment]
Should we add some CI check or step to make sure all PRs generate the new file? Or the generate be run on `master`?

File: README.md [hunk 1]
```diff
@@ -12,33 +14,35 @@ A list of active (see [active](#active) for how we define active) declarative pr
...
 - [cvladfp](https://twitch.tv/cvladfp):
   - Speaking: EN
-  - Programming Languages: Haskell, Purescript
+  - Programming Languages: Haskell, PureScript
   - Schedule: Mon, Tue, Thurs, Wed, Fri, Sun at 8PM UTC
 - [gillchristian](https://twitch.tv/gillchristian):
   - Speaking: EN
-  - Programming Languages: TypeScript (fp-ts), Haskell, Purescript, Reason, Rust
-  - Schedule: Thursdays 6:00 PM UTC ([gillchristian.yxz/stream](https://gillchristian.xyz/stream))
+  - Programming Languages: Haskell, Purescript, TypeScript (fp-ts), Reason, Rust
```
Comment (chiroptical) [inline README.md:25]: s/Purescript/PureScript

**Review by chiroptical — COMMENTED** [review]
LGTM

**Comment by chiroptical** [comment]
I think it would be ideal if generation of the README happened at commit time. We could add a nix build that handles the pre-commit action? This would require a contributor to use nix which could be viewed as a barrier. Let me know what you think

File: README.md (hunk 1 above)
Comment (gillchristian) [reply README.md]: Good catch

**Comment by gillchristian** [comment]
Sounds good. I personally don't mind having to install a tool. But if it's a blocker for someone we can always do it for them.

The only problem is I haven't done anything with nix besides running pre-configured `nix-shell` :sweat_smile:

**Comment by chiroptical** [comment]
I think this is good to merge now. I'll work on a pre-commit nix build to auto generate the README. I would also like a CI action to check that the two documents are the same.

**Pull request merged**